package.json

```diff
@@ -14,7 +14,7 @@
     "debugger"
   ],
   "engines": {
-    "vscode": "^1.22.0"
+    "vscode": "^1.32.0"
```

Comment:
When is the features introduced in vscode for additional icons in debug toolbar?

Reply:
yes, 1.32 add the support to extend the debug toolbar buttons.